.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: ruby
 rvm:
-  - '2.0'
   - '2.1'
   - '2.2'
-  - 'jruby-19mode'
+  - 'jruby-9000'
```

Comment:
Aw, just missed it

![it__s_over_9000_](https://cloud.githubusercontent.com/assets/2230482/7262680/2113345a-e841-11e4-9f9d-5ea42c1d0cc6.png)